packages/app/src/client/services/AdminHomeContainer.js

```diff
@@ -44,19 +55,113 @@ export default class AdminHomeContainer extends Container {
       const response = await this.appContainer.apiv3.get('/admin-home/');
       const { adminHomeParams } = response.data;
 
-      this.setState({
+      this.setState(prevState => ({
+        ...prevState,
         growiVersion: adminHomeParams.growiVersion,
         nodeVersion: adminHomeParams.nodeVersion,
         npmVersion: adminHomeParams.npmVersion,
         yarnVersion: adminHomeParams.yarnVersion,
         installedPlugins: adminHomeParams.installedPlugins,
         envVars: adminHomeParams.envVars,
-      });
+      }));
     }
     catch (err) {
       logger.error(err);
       toastError(new Error('Failed to fetch data'));
     }
   }
 
+  /**
+   * used to set button text when copying bug report
+   */
+  onCopyBugReport() {
+    this.setState(prevState => ({
+      ...prevState,
+      copyState: this.copyStateValues.DONE,
+    }));
+
+    this.timer = setTimeout(() => {
+      this.setState(prevState => ({
+        ...prevState,
+        copyState: this.copyStateValues.DEFAULT,
+      }));
+    }, 500);
+  }
+
+  /**
+   * generates bug report with prefilled system information
+   */
+  generatePrefilledBugReport() {
+    return `Environment
+------------
+
+### Host
+
+| item     | version |
+| ---      | --- |
+|OS        ||
+|GROWI     |${this.state.growiVersion}|
+|node.js   |${this.state.nodeVersion}|
+|npm       |${this.state.npmVersion}|
+|yarn      |${this.state.yarnVersion}|
+|Using Docker|yes/no|
+|Using [growi-docker-compose][growi-docker-compose]|yes/no|
+
+[growi-docker-compose]: https://github.com/weseek/growi-docker-compose
+
+*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*
```

Comment:
I think the text that will be copied by CopyToClipboard should be enough only these lines I marked.

Reply:
I've made the changes you requested. @yuki-takei

Screenshot:
![image](https://user-images.githubusercontent.com/34886045/131471466-30d97c31-5331-48ac-8e7d-fb67380664dd.png)


My clipboard now holds:
```md
| item     | version |
| ---      | --- |
|OS        ||
|GROWI     |4.3.3-RC|
|node.js   |14.17.4|
|npm       |6.14.14|
|yarn      |1.22.5|
|Using Docker|yes/no|
|Using [growi-docker-compose][growi-docker-compose]|yes/no|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*
```

## Note
Translations still need to be done.